pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -606,6 +617,49 @@ func (r *ReconcileHawtio) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{Requeue: true}, nil
 	}
 
+	// Reconcile console link in application menu
+	if isClusterDeployment {
+		consoleLinkCrd := &apiextensionsv1beta1.CustomResourceDefinition{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Name: consoleLinkCrdName}, consoleLinkCrd)
+		if err != nil && errors.IsNotFound(err) {
+			reqLogger.Info("No support for console links")
+		} else if err != nil {
+			reqLogger.Error(err, "Failed to get ConsoleLink CRD")
+		} else {
+			hawtconfig, err := osutil.GetHawtconfig(configMap)
+			if err != nil {
+				reqLogger.Error(err, "Failed to get hawtconfig")
```

Comment:
The error should be re-enqueued. Then the `else` branch below can be removed.

Reply:
Done

Reply:
Done